File: 프로그래머스/0/120903. 배열의 유사도/배열의 유사도.cpp
#include <string>
#include <vector>

using namespace std;

int solution(vector<string> s1, vector<string> s2) {
    int answer = 0;
    for(int i=0; i<s1.size(); i++) {
        for(int j=0; j<s2.size(); j++) {
            if(s1[i] == s2[j]) answer++;
        }
    }
    return answer;
}

File: 프로그래머스/0/120903. 배열의 유사도/배열의 유사도.swift
import Foundation

func solution(_ s1:[String], _ s2:[String]) -> Int {
    var cnt: Int = 0
    
    for i in s1 {
        for j in s2 {
            if i == j {
                cnt += 1
            }
        }
    }
    
    return cnt
}